src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/OpphørValideringService.kt

```diff
@@ -49,15 +49,13 @@ class OpphørValideringService(
     }
 
     fun validerVedtaksperioderAvkortetVedOpphør(
-        forrigeBehandlingsVedtaksperioder: List<Vedtaksperiode>,
+        forrigeBehandlingsVedtaksperioder: List<Periode<LocalDate>>,
         revurderFraDato: LocalDate,
     ) {
```

Comment:
Føler det er litt tydligere når denne metode tar inn Vedtaksperiode i stedet for Periode<LocalDate>
Kan man ev legge til en metode:

```kotlin
fun validerVedtaksperioderAvkortetVedOpphør(
        forrigeBehandlingsVedtaksperioder: List<no.nav.tilleggsstonader.sak.vedtak.læremidler.domain.Vedtaksperiode>,
        revurderFraDato: LocalDate,
    ) {
        validerVedtaksperioderAvkortetVedOpphør(forrigeBehandlingsVedtaksperioder.map { it.tilFellesDomeneVedtaksperiode() }, revurderFraDato)
    }
```

Og endre den eksisterende til å bruke `no.nav.tilleggsstonader.sak.vedtak.domain.Vedtaksperiode`

Vi har en annen oppgave på at vi skal bruke samme vedtaksperiode for alle våre stønader. Når den gjøres vil man kunne ta bort den metoden som bruker `læremidler.domain.Vedtaksperiode`

Reply:
Hmm, får visst en Platform declaration clash pga lik JVM-signatur nå 🤔